src/routes/load/components/ReviewInformation/index.tsx

```diff
@@ -104,7 +104,7 @@ const ReviewComponent = ({ userAddress, values }: Props): ReactElement => {
           </Block>
           <Hairline />
           {owners.map((address, index) => (
-            <>
+            <Fragment key={`${address}_${index}`}>
```

Comment:
`index` doesn't seem to be necessary, `address` should be enough as they must be unique at this point. Isn't it?